app/policies/affiliate_policy.rb

```diff
@@ -0,0 +1,23 @@
+class AffiliatePolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      if user.admin? || user.legal?
+        scope.all
+      else
+        scope.none
+      end
+    end
+  end
+
+  def index?
+    user.admin? || user.legal?
```

Comment:
Overlap between admin and legal in a lot of these policies, maybe we should move this to be a `user.manage_contracts?` that does this instead. Feel free to change up the name to something else however.

Reply:
I still haven't done this. Feel free to either merge it, or otherwise we can fix this tomorrow because I have some questions about it:)